beacon_node/operation_pool/src/lib.rs

```diff
@@ -562,6 +566,42 @@ impl<T: EthSpec> OperationPool<T> {
         )
     }
 
+    /// Get a list of execution changes to be broadcast at the Capella fork.
+    ///
+    /// The list that is returned will be shuffled to help provide a fair
+    /// broadcast of messages.
+    pub fn get_bls_to_execution_changes_for_received_pre_capella(
```

Comment:
```suggestion
    pub fn get_bls_to_execution_changes_received_pre_capella(
```

Reply:
Ah yep, good catch. This was a fumbled find-and-replace!

Reply:
Fixed in https://github.com/sigp/lighthouse/pull/3919/commits/f45a1c87db58d0de9d38a09f6ccfe712c73a71d3